app/routes/events_.$eventId.tsx

```diff
@@ -0,0 +1,73 @@
+import type { LoaderFunction } from '@remix-run/node';
+import { useParams, useLoaderData, Link } from '@remix-run/react';
+import { db } from '~/modules/database/db.server';
+import { json } from '@remix-run/node';
+import { eventDataPatcher } from '~/modules/db_utils/event';
+
+export const loader: LoaderFunction = async ({ params }) => {
+  const event = await db.event.findFirstOrThrow({
```

Comment:
NIT: let's maybe use just `findUnique` or `find` and if it's `null`, we could throw a 404 response to trigger the 404 page!